src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXOverrideMismatch/PXOverrideMismatchAnalyzer.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Utilities.Roslyn.Semantic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Linq;
+using Acuminator.Analyzers.StaticAnalysis.AnalyzersAggregator;
+using Acuminator.Utilities;
+using Acuminator.Utilities.DiagnosticSuppression;
+
+namespace Acuminator.Analyzers.StaticAnalysis.PXOverrideMismatch
+{
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class PXOverrideMismatchAnalyzer : SymbolAnalyzersAggregator<IPXGraphAnalyzer>
```

Comment:
@taskinkemal this is incorrect design for the analyzer. The `SymbolAnalyzersAggregator` should be used only of you are looking to aggregate several analyzers that can share a common state between them such as graph and dac analyzers we have. That's why we have a notion of the "aggregated" analyzers that are not actually derived from the Roslyn base analyzer class. 

In case of this diagnostic, if you want to check the `PXOverride` signature you need to analyze only graph extensions and we have a perfect base class for that. You should derive from the `PXGraphAggregatedAnalyzerBase` and implement abstract base classes. 

Take a look at `CallingBaseActionHandlerFromOverrideHandlerAnalyzer` and `NoIsActiveMethodForExtensionAnalyzer` analyzers for an example

Reply:
Ok, this is done. Thank you for your feedback. Please review again.